styles.css

```diff
@@ -0,0 +1,807 @@
+  height: 20rem;
+  align-items: center;
+  top: 0;
+}
+
+.about-img2,
+.about-img1 {
+  position: relative;
+  background: #000 url(img/system4.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 0 4rem;
+  color: #fff;
+  height: 17rem;
+  width: 30rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.5rem;
+  text-align: center;
+  max-width: 100%;
+}
+
+.about-img2 > div:hover {
+  display: none;
+}
+
+.about-img1 > div:hover {
+  display: none;
+}
+
+.color-overlay {
+  width: 100%;
+  height: 100%;
+  background: red;
+  opacity: 0.5;
+  position: absolute;
+}
+
+.pro p {
+  font-size: 0.8rem;
+  margin: 0 1rem 0.5rem 1rem;
+  text-align: center;
+}
+
+.about-img-courses li p {
+  z-index: 1;
+  font-weight: bold;
+}
+
+.para {
+  position: relative;
+  border: 2px solid white;
+  max-width: 45rem;
+  line-height: 1.7;
+  margin-top: 1rem;
+  padding: 2rem;
+  background-color: #c4c4c4;
+  font-weight: 600;
+}
+
+.showcase h2 {
+  position: relative;
+  font-weight: 800;
+  padding-top: 2rem;
+  letter-spacing: 0.2rem;
+}
+
+.para1 {
+  position: relative;
+  max-width: 100%;
+  font-weight: 600;
+  line-height: 1.7;
+  margin-top: 1rem;
+  padding: 2rem;
+  color: #e85242;
+  font-size: 2rem;
+  padding-left: 0;
+}
+
+.program {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  background-color: #262930;
+  color: #fff;
+  padding: 1rem 0 0 0;
+}
+
+.pro h5 {
+  color: #ec4b3a;
+  font-size: 1rem;
+  font-weight: bold;
+  padding-top: 0.5rem;
+}
+
+.courses {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  margin: 0 3rem;
+  max-width: 70%;
+}
+
+.pro {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #3d4046;
+  margin: 0 0.1rem;
+}
+
+.img-logo {
+  max-width: 20%;
+  align-self: center;
+  padding-top: 1rem;
+}
+
+hr {
+  background-color: rgb(250, 19, 19);
+  font-weight: bolder;
+  width: 3rem;
+}
+
+.pro:hover {
+  border: 1px solid #fff;
+}
+
+u {
+  padding: 1rem 0 5rem 0;
+}
+
+.footer1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #414246;
+  color: #fff;
+  padding: 2.5rem;
+  max-width: 100%;
+}
+
+.footer-logos {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer-logos img {
+  margin: 0 0.5rem;
+}
+
+.footer2-img img {
+  max-width: 30%;
+}
+
+.footer-about {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #2e2e2e;
+  padding: 3.5rem 0;
+  max-width: 100%;
+  color: #fff;
+}
+
+.footer2 {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #d7d7d7;
+  max-width: 100%;
+}
+
+.footer2-img {
+  display: flex;
+  align-items: center;
+  margin: 0.5rem 0 0 1rem;
+  max-width: 100%;
+}
+
+.footer-desc {
```

Comment:
- [ ] There are still significant variances between your footer and the provided design template. Please refer to the last two reviewers comments and the screenshot provided. The wide margin between the logo and the text still persists and the color schemes are still different. Please, let's fix it.